packages/noports_core/lib/src/sshnp/models/sshnp_arg.dart

```diff
@@ -346,4 +350,44 @@ class SshnpArg {
     negatable: false,
     parseWhen: ParseWhen.commandLine,
   );
+  static const authenticateClientToRvdArg = SshnpArg(
+    name: 'authenticate-client-to-rvd',
+    help: 'When false, client will not authenticate itself to rvd',
+    defaultsTo: DefaultArgs.authenticateClientToRvd,
+    format: ArgFormat.flag,
+    parseWhen: ParseWhen.commandLine,
+    mandatory: false,
+  );
+  static const authenticateDeviceToRvdArg = SshnpArg(
+    name: 'authenticate-device-to-rvd',
+    help: 'When false, device will not authenticate to the socket rendezvous',
+    defaultsTo: DefaultArgs.authenticateDeviceToRvd,
+    format: ArgFormat.flag,
+    parseWhen: ParseWhen.commandLine,
+    mandatory: false,
+  );
+  static const encryptRvdTrafficArg = SshnpArg(
+    name: 'encrypt-rvd-traffic',
+    help: 'When true, traffic via the socket rendezvous is encrypted,'
+        ' in addition to whatever encryption the traffic already has'
+        ' (e.g. an ssh session)',
+    defaultsTo: DefaultArgs.encryptRvdTraffic,
+    format: ArgFormat.flag,
+    parseWhen: ParseWhen.commandLine,
+    mandatory: false,
+  );
+  static const discoverDaemonFeaturesArg = SshnpArg(
```

Comment:
would be nice to have an abbreviation for this as well.

Reply:
Have added as 'F'
Have also added abbreviation 'E' for 'encrypt-rvd-traffic'